test/rounding.jl

```diff
@@ -434,10 +437,11 @@ const native_rounding_modes = (
 
 # Checks that each rounding mode is faithful.
 @testset "IEEEFloat(::BigFloat) faithful rounding" begin
-    for pr ∈ 1:200
+    for pr ∈ fib200
         setprecision(BigFloat, pr) do
             exp = 500
-            bf_samples = float_samples(BigFloat, (-exp):exp, 20)
+            bf_samples = float_samples(BigFloat, (-exp):exp, 20) # about 40040 random values
+            @show length(bf_samples)
```

Comment:
Accident?

Reply:
yeah, apparently committed at the wrong time